package.json

```diff
@@ -97,18 +98,17 @@
     "style-loader": "^0.20.2",
     "traverse": "^0.6.6",
     "uglify-js": "^3.4.6",
-    "uuid": "^3.2.1"
-  },
-  "devDependencies": {
```

Comment:
Something went wrong here. you removed the devDependencies

Reply:
so I pushed them up into the `dependencies` for the sake of the buildpack on PWS.  I have an open issue with them about why they can't see devDeps when packaging.  Moving them makes it so the person deploying this app only has to run a single command with a single CLI to get Nightscout deployed.  Leaving webpack in the devDeps section would require them to run an `npm install` before deploying.  I was attempting to make it as easy as possible on the user.

Reply:
What we need to do is publish the npm package with the generated webpack javscript included and use that for these kind of stuff. Current version requires `npm install` which isn't very strange. Isn't there a way to let the cf install execute the `npm install`?

Reply:
Is the symlink problem a pivotal problem? We can't change our dependencies for a bug in the deployment system of a hosting provider. We need to find/implement a better solution.

Reply:
I think it is a buildpack issue, but I haven't gotten a response from that issue yet either.  There is a workaround that would require an `npm install` be run before the push.  I will change the dev dependencies back and add that section to the new PWS README.

Reply:
@thadc23 please let me know if you can find a solution. I'm planning on publishing a cgm-remote-monitor npm for 0.12 and then we don't need the devDependencies because they will be used with the npm packaging.